### DISCUSSION TITLE
How to sort found items as newest first (file search)

### DISCUSSION THREAD

**Discussion by NolanNolan** [Flow-Launcher/Flow.Launcher#1436] [Q&A]
Hi

How to sort found items as newest first when using file search ?

Ie say i search for content in docx files, how do i make the result appear as the newest first ?

thanks

Nolan

**Comment by taooceros** [comment]
Currently we haven't supported yet, but maybe in the future we would like to support sort option.

**Comment by NolanNolan** [comment]
HI

ok how are results then sorted as it is now ?

Nolan

**Reply by taooceros** [reply]
By name currently. The everything plugin has the sort option, and we are planning to merge them.

**Comment by NolanNolan** [comment]
Ok now i installed everything in order to make the list sort returned items by dates last saved. 

But is it possible to set "everything" to be used aleasy without a actionkeyword ?
that way i guess i can have listeditems listed after date ?

There seem to no pattern in the sorting order when using the standard search see attached screnshot, i search for Pensionsberegning 31-12-2021 inkl budget 2022.xlsx but the file beginning with Pensionsberegning is not the first listed hence its not alphabetic - correct?

Looks like th sort order is random.  Coul dbe changed to be sorted after last saved ie date ? Or an option to choose sort order of eg name og last saved etc ?

![image](https://user-images.githubusercontent.com/30736282/193456403-afa7358b-aeeb-4447-8794-62daca498c23.png)



thanks 👍
![Clipboard Image](https://user-images.githubusercontent.com/30736282/193456428-094f5494-db4c-42a6-96c6-4cecbdfafcde.jpg)

**Reply by taooceros** [reply]
Maybe you want to change the actionkeyword of explorer plugin? It is possible that these results are returned not in the order because they are from explore plugin.

**Comment by NolanNolan** [comment]
Yes could be. But what do you get as sort order when using exolorer plugin?

**Reply by taooceros** [reply]
By Name for now. I would like to port the sort option from everything but I will leave it in the future (probably will be included in 1.10).

**Comment by NolanNolan** [comment]
Thanks, would be really useful, looking forward to this feature 🙂👍

Hilsen Jørn

On 9. oktober 2022 18.30.22 Kevin Zhang ***@***.***> wrote:

By Name for now. I would like to port the sort option from everything but I will leave it in the future (probably will be included in 1.10).
